src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -102,6 +102,11 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
               useMarkdown={true}
             />
           </Grid>
+          <Grid item xs={12}>
```

Comment:
Hmm, logically I don't think I would put the DUOS ID at the top of our metadata grid, above even the root dataset and snapshot ID.

We can check with Lou about ordering, but I'd vote for putting this after PHS ID and Consent Code, since these are all fields which impact snapshot access.

Reply:
Root dataset > snapshot id > duos id

Is an order that makes sense to me too! If duos id is only used for modifying user access, i think we could move more of the data around in the summary view - Maybe we could start adding section headers, so we can have

Snapshot information: id, date created, billing profile id
Snapshot data metadata: root dataset, storage, google data project
Snapshot access: PHS ID, consent code, duos ID

?

Reply:
I like where you're going with this!  We've exposed quite a bit of metadata over the last few months and I agree that there's an opportunity for more logical grouping.  But I'm going to approve this PR as it stands, it would be better to get to check with Lou and the rest of the team about the subsections and handle separately.

Reply:
Sounds good! :)